client/stylesheets/_toCat.scss

```diff
@@ -2973,17 +2973,13 @@ i:hover {
 
 .slider label {
     /*Lets add some spacing for the thumbnails*/
-    margin: 18px 0 0 18px;
+    margin: 9px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border, border-radius, cursor, float, margin